### DISCUSSION TITLE
表格重新加载数据后，上一次勾选的状态保留，而且手动清除勾选状态也只能清除当前页的，其它页面无效

### DISCUSSION THREAD

**Discussion by HuangLei0x2e5** [layui/layui#2023] [问答]
当我重新加载表格数据时，上一次勾选的行还是保留着被勾选状态，我希望数据更新时会自动清除掉上一次的选中状态

我只好手动添加了一个操作：
          table.setRowChecked('ap_table', {index: 'all',checked: false});
但是这也只是清掉了当前页的勾选，其它页面还是保留着勾选状态

**Comment by sentsim** [comment]
Layui 不会记忆选中状态，每次翻页或者重载数据时，都只会根据数据的 `LAY_CHECKED` 值进行勾选标注。
建议你检查下别的业务代码是否单独做了勾选记忆功能。

也可与官网示例进行对比：https://layui.dev/docs/2/table/

**Comment by HuangLei0x2e5** [comment]
我找到原因了
为了减轻后端的压力，我是在前端缓存的数据里做的筛选，但是前端拿到的数据被layui塞进了LAY_CHECKED 等几个属性，我无论如何筛选这几个数据，LAY_CHECKED是不会变的

**Reply by LSL1618** [reply]
暂存前端缓存的时候做一下属性过滤即可，比如删除`LAY_CHECKED`属性，这样后面使用缓存数据的时候就不会被干扰！
